File: Exercice_1/moyenne.py
def calculer_moyenne(tableau):
    if not tableau:
        return 0  # ou éventuellement lever une exception
    return sum(tableau) / len(tableau)

# Exemple d'utilisation
nombres = [4, 8, 15, 16, 23, 42]
moyenne = calculer_moyenne(nombres)
print(f"La moyenne est : {moyenne}")

File: Exercice_1/moyenne.kt
fun calculerMoyenne(tableau: IntArray): Double {
    if (tableau.isEmpty()) return 0.0
    return tableau.sum().toDouble() / tableau.size
}

// Exemple d'utilisation
fun main() {
    val nombres = intArrayOf(4, 8, 15, 16, 23, 42)
    val moyenne = calculerMoyenne(nombres)
    println("La moyenne est : $moyenne")
}
